demo/src/app.js

```diff
@@ -160,17 +160,18 @@ const styles = (theme) => ({
         cursor: 'initial',
     },
     tableCellColor: {
-        color: 'blue',
+        color: theme.palette.primary.contrastText,
     },
     header: {
-        backgroundColor: 'lightblue',
+        backgroundColor: theme.palette.primary.light,
+        color: theme.palette.primary.contrastText,
         fontWeight: 'bold',
     },
     rowBackgroundDark: {
-        backgroundColor: '#81BEF7',
+        backgroundColor: theme.palette.primary.dark,
     },
     rowBackgroundLight: {
```

Comment:
Why do we have to classes here ? should be only one set to theme.palette.primary.main and change when switching the theme no ?